Source/Fuse.Models/ModelJavaScript.uno

```diff
@@ -80,6 +97,39 @@ namespace Fuse.Models
 			rootVisualProvider.Root.Children.Add(appModel);
 		}
 
+		static IExpression[] UnpackArgs(IExpression argsExpr)
+		{
+			var vector = argsExpr as Reactive.Vector;
```

Comment:
IS this guaranteed to only trigger if the expression is something like  `a,b,c` instead of say a single value `list` that happens to be a vector?

Reply:
I assume you are referring to the case where I have a binding (like `ModelArgs="{list}"`), where the binding would resolve to a `Fuse.Reactive.Vector`? If so, I don't see how that could be the case. Since `SetModelArgs` takes an `IExpression`, the UX compiler will provide it with the expression tree rather than the result of evaluating the expression. In the case of `{list}`, that would become a `new Fuse.Reactive.Data("list")`, the dynamic cast would evaluate to `null`, and we'd hit the other case, where we return an array of just that one expression.

Reply:
Oh, I guess you mean something like `ModelArgs="list"` where `list` is a `Fuse.Reactive.Vector` that is in scope? I'm fairly certain  that would become a `new Fuse.Reactive.Constant(list)`, but I'm not 100% sure. I'll have to test this a bit further.

Reply:
Yes, those two caess, where `{list}` binds to some kind of vector type. But I guess you're correct, you're checking the actual IExpression type, not the resulting type.

The other case `ModelArgs="list"` where `list` is a global would be interesting to test. I don't think it'd come up much, but I think we'd be safer if we had some tests covering these cases.

Reply:
Indeed. I'll get a test in for that.

Reply:
@mortoray Yes, given the following UX:

```
<Fuse.Reactive.Vector ux:Name="vec" ux:AutoBind="false" />
<Panel Model="UX/Args.VectorInScope" ModelArgs="vec" />
```

The UX compiler will generate something like the following (cleaned up for brevity)

```
vec = new global::Fuse.Reactive.Vector();
var temp = new global::Fuse.Controls.Panel();
var temp1 = new global::Fuse.Reactive.Constant(vec);
global::Fuse.Models.ModelJavaScript.SetModelArgs(temp, temp1);
```